tools/peview/peexceptprp.c

```diff
@@ -166,7 +166,80 @@ VOID PvEnumerateExceptionEntries(
     }
     else if (imageMachine == IMAGE_FILE_MACHINE_ARM64)
     {
-        /* Todo */
+        for (ULONG i = 0; i < exceptions.NumberOfEntries; i++)
+        {
+            PPH_IMAGE_RUNTIME_FUNCTION_ENTRY_ARM64 entry = PTR_ADD_OFFSET(exceptions.ExceptionDirectory, i * sizeof(PH_IMAGE_RUNTIME_FUNCTION_ENTRY_ARM64));
+            INT lvItemIndex;
+            PPH_STRING symbol;
+            PPH_STRING symbolName = NULL;
+            WCHAR value[PH_INT64_STR_LEN_1];
+            IMAGE_ARM64_RUNTIME_FUNCTION_ENTRY_XDATA* exceptionData;
+
+            PhPrintUInt64(value, ++count);
+            lvItemIndex = PhAddListViewItem(ListViewHandle, MAXINT, value, entry);
+
+            PhPrintPointer(value, UlongToPtr(entry->BeginAddress));
+            PhSetListViewSubItem(ListViewHandle, lvItemIndex, 1, value);
+
+            ULONG functionLength = entry->FunctionLength << 2;
+            if (entry->Flag != 0) //Packed Unwind Data
```

Comment:
Is it worth showing the type of packing here (instead of yes/no), the bits are described here:
https://learn.microsoft.com/en-us/cpp/build/arm64-exception-handling?view=msvc-170#packed-unwind-data

Reply:
To be clear, instead of showing L"Yes" for "Packed", we could instead show something more descriptive:

```
    00 = packed unwind data not used; remaining bits point to an .xdata record
    01 = packed unwind data used with a single prolog and epilog at the beginning and end of the scope
    10 = packed unwind data used for code without any prolog and epilog. Useful for describing separated function segments
    11 = reserved.
```

Reply:
Any idea on column names and what text for each case?